### PR TITLE
Php 7 support

### PR DESCRIPTION
**Pull request by chrisryan**
Updated version to include Php 7.
Removed unused util package.

Not sure what other packages currently use this and there are no tests to verify otherwise. Will need some testing.

### CODE REVIEW

**Comment by chrisryan** [comment]
Please hold off from merging this until I have a chance to update/test the tool that is referencing this package.

File: composer.json [hunk 1]
```diff
@@ -16,8 +16,7 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "~5.4",
-        "dominionenterprises/util": "~1.0",
+        "php": "~5.4 || ~7.0",
```
Comment (chadicus) [inline composer.json]: Shouldn't this be `^5.6 || ^7.0` not `5.4`

**Comment by chrisryan** [comment]
I had left this one 5.4 intentionally but at this time I think we can update to 5.6 to be consistent.

File: composer.json (hunk 1 above)
Comment (chrisryan) [inline composer.json]: I had left this one 5.4 intentionally but at this time I think we can update to 5.6 to be consistent.

**Comment by chrisryan** [comment]
Updated

**Pull request merged**